CONTRIBUTING.md

```diff
@@ -20,9 +26,14 @@ See Branching overview below - We use `dev` for our current development.
 
 In your fork from the root of the monorepo:
 
-- `npm install` - this will install Lerna at the root and allow the remaining scripts to be called.
-- (optional) `npm run install-git-hooks` - installs a pre-push hook that will lint all samples and apps before a `git-push`. Opt out per-push with the `--no-verify` flag.
-- `npm run reset` -> will clean all `node_modules`, re-install them with `lerna bootstrap`, and then build all JSS packages.
+- `yarn install` - this will install Lerna at the root, packages, samples and will link them and allow the remaining scripts to be called.
+- (optional) `yarn install-git-hooks` - installs a pre-push hook that will lint all samples and apps before a `git-push`. Opt out per-push with the `--no-verify` flag.
+- `yarn reset` -> will clean all `node_modules`, re-install dependencies and link them, and then build all JSS packages.
```

Comment:
For azure-pipelines we should have a separate script to build packages, we can't use `reset` there. I would rather have a separate script for building packages

Reply:
Not saying we don't need a dedicated build script - saying we should actually use that script here (instead of `reset`). `reset` is redundant / overkill for the initial setup. Still totally valid for the Troubleshooting section below.
```suggestion
- `yarn build` -> will build all JSS packages.
```

Reply:
Now I got your idea. Changed it, and I left description for `reset` command but moved it to the bottom